_observing-your-data/query-insights/live-queries.md

```diff
@@ -0,0 +1,107 @@
+
+## Endpoints
+
+```json
+GET /_insights/live_queries
+```
+
+## Query parameters
+
+The following table lists the available query parameters. All query parameters are optional.
+
+| Parameter | Data type | Description |
+| :--- | :--- | :--- |
+| `verbose` | Boolean | Whether to include detailed query information in the output. Default is `true`. |
+| `nodeId` | String | A comma-separated list of node IDs to filter the results. If omitted, queries from all nodes are returned. |
+| `sort` | String | The metric to sort the results by. Valid values are `latency`, `cpu`, `memory`. Default is `latency`. |
+| `size` | Integer | The number of query records to return. Default is 100. |
+
+## Example request
+
+The following example request fetches top 10 queries sorted by CPU usage, with verbose output disabled:
+
+```json
+GET /_insights/live_queries?verbose=false&sort=cpu&size=10
+```
+{% include copy-curl.html %}
+
+## Example response
+
+```json
+{
+  "live_queries" : [
+    {
+      "timestamp" : 1745359226777,
+      "id" : "troGHNGUShqDj3wK_K5ZIw:512",
+      "description" : "indices[my-index-*], search_type[QUERY_THEN_FETCH], source[{\"size\":20,\"query\":{\"term\":{\"user.id\":{\"value\":\"userId\",\"boost\":1.0}}}}]",
+      "node_id" : "troGHNGUShqDj3wK_K5ZIw",
+      "measurements" : {
+        "latency" : {
+          "number" : 13959364458,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        },
+        "memory" : {
+          "number" : 3104,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        },
+        "cpu" : {
+          "number" : 405000,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        }
+      }
+    },
+    {
+      "timestamp" : 1745359229158,
+      "id" : "Y6eBnbdISPO6XaVfxCBRgg:454",
+      "description" : "indices[my-index-*], search_type[QUERY_THEN_FETCH], source[{\"size\":20,\"query\":{\"term\":{\"user.id\":{\"value\":\"userId\",\"boost\":1.0}}}}]",
+      "node_id" : "Y6eBnbdISPO6XaVfxCBRgg",
+      "measurements" : {
+        "latency" : {
+          "number" : 11579097209,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        },
+        "memory" : {
+          "number" : 3104,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        },
+        "cpu" : {
+          "number" : 511000,
+          "count" : 1,
+          "aggregationType" : "NONE"
+        }
+      }
+    }
+  ]
+}
+```
+
+## Response fields
+
+| Field               | Data type | Description                                                                                                |
+| :------------------ | :-------- | :--------------------------------------------------------------------------------------------------------- |
+| `timestamp`         | Long      | The time the query task started, in milliseconds since the epoch.                                          |
```

Comment:
```suggestion
| `timestamp`         | Long      | The time at which the query task started, in milliseconds since the epoch.                                          |
```